test/Microsoft.TestPlatform.TestUtilities/TestUtils.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.TestPlatform.TestUtilities;
+
+public static class TestUtils
```

Comment:
This name seems soo generic that this class will become another "IntegrationTestBase" with thousands of lines of unrelated code. 

Maybe rename it to OSUtils?